File: server/Models/Picture.cs
using System.ComponentModel.DataAnnotations;

namespace post_it_dotnet_fullstack.Models;

public class Picture
{
  [Url, MaxLength(2000)] public string ImgUrl { get; set; }
  public string CreatorId { get; set; }
  public int AlbumId { get; set; }
  public Profile Creator { get; set; }
}

File: client/src/models/Picture.js
import { Profile } from "./Account.js"

export class Picture {
  constructor(data) {
    this.id = data.id
    this.createdAt = new Date(data.createdAt)
    this.updatedAt = new Date(data.updatedAt)
    this.imgUrl = data.imgUrl
    this.albumId = data.albumId
    this.creatorId = data.creatorId
    this.creator = new Profile(data.creator)
  }
}